crates/fuel-core/src/graphql_api/view_extension.rs

```diff
@@ -37,7 +39,15 @@ impl Extension for ViewExtension {
         next: NextPrepareRequest<'_>,
     ) -> ServerResult<Request> {
         let database: &ReadDatabase = ctx.data_unchecked();
-        let view = database.view();
+        let view = database.view().map_err(|e| {
+            ServerError::new(
+                e.to_string(),
+                Some(Pos {
+                    line: line!() as usize,
+                    column: column!() as usize,
+                }),
```

Comment:
This feels weird. Maybe it's actually supposed to work like this, but the mismatch between `line!` and `column!` evaluated separately seems wrong.